scalingo/data_scalingo_container_size.go

```diff
@@ -0,0 +1,103 @@
+package scalingo
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/Scalingo/go-scalingo/v5"
+)
+
+func dataSourceScContainerSize() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceScContainerSizeRead,
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"sku": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"human_name": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"human_cpu": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"memory": {
+				Type:     schema.TypeInt,
+				Computed: true,
+			},
+			"pids_limit": {
+				Type:     schema.TypeInt,
+				Computed: true,
+			},
+			"hourly_price": {
+				Type:     schema.TypeInt,
+				Computed: true,
+			},
+			"thirtydays_price": {
+				Type:     schema.TypeInt,
+				Computed: true,
+			},
+			"swap": {
+				Type:     schema.TypeInt,
+				Computed: true,
+			},
+			"ordinal": {
+				Type:     schema.TypeInt,
+				Computed: true,
+			},
+		},
+	}
+}
+
+// dataSourceScNotificationPlatformRead performs the Scalingo API lookup
```

Comment:
The comment is rather useless. It does not explain anything. A lookup of what? For what?

```suggestion
// dataSourceScContainerSizeRead performs the Scalingo API lookup
```